src/encode.c

```diff
@@ -5766,6 +5766,7 @@ find_prev_entity (Dwg_Object *obj)
   Dwg_Data *dwg = obj->parent;
   if (obj->supertype != DWG_SUPERTYPE_ENTITY)
     return NULL;
+  /*
   for (BITCODE_BL i = obj->index - 1; i > 0; i--)
```

Comment:
They are already searched backwards. Are there really so many non-entities? well, there are at the end, but still curious about the speedup.

Reply:
lsearch
100000 entities used 00:03:34
300000 entities used 00:33:53

bsearch
100000 entities Used 00:01:19
300000 entities used  00:14:53

bsearch+prev_entity

100000 entities used 00:00:02
300000 entities used 00:00:23